CONTRIBUTING.md

```diff
@@ -176,3 +176,12 @@ If you use VSCode, you can place the following in a `settings.json` file in the
 See [Superchain Upgrades] OP Stack specifications.
 
 [Superchain Upgrades]: https://specs.optimism.io/protocol/superchain-upgrades.html
+
+## CircleCI Checks
+
+The following CircleCI checks are required and must pass before submitting a pull request:
+
+- `ci/circleci: compute-genesis-diff` — Your tests passed on CircleCI!
+- `ci/circleci: compute-rollup-config-diff` — Your tests passed on CircleCI!
```

Comment:
Did you mean to past the "Your tests passed" bit?

Reply:
nope